frontend/src/components/AddRecipeForm.js

```diff
@@ -99,9 +122,9 @@ const AddRecipeForm = () => {
           {...formik.getFieldProps('description')}
         />
         {formik.touched.description && formik.errors.description ? (
-          <div>{formik.errors.description}</div>
+          <div className="text-danger">{formik.errors.description}</div>
```

Comment:
Ezen a ponton érdemes átgondolni, hogy nem lenne-e jobb kiszervezni egy komponensbe az input mezőket labellel együtt és ott egy useField hookal általánosítani és csak a name-et lepasszolni neki, hogy tudja mégis melyik mezőhöz tartozik.

Reply:
Esetleg lehet egy required prop-ja is, hogy tudja, hogy kötelező mező-e és ettől függően mehet a csillagozás.

Reply:
https://formik.org/docs/api/useField nézegettem és az elméletit is de hardcore ahhoz képest , ha Szerdáig várhat amikor bent leszel , szívesen ránéznék

Reply:
Ok, akkor ebben a PR-ben maradjon így és szerdán nyitunk rá egy külön issue-t. Majd PM-nek jelezzétek, hogy kellene egy technical story.